geoh5py/ui_json/ui_json.py

```diff
@@ -71,8 +71,16 @@ class BaseUIJson(BaseModel):
     monitoring_directory: OptionalPath
     conda_environment: str
     workspace_geoh5: OptionalPath | None = None
+    _path: Path | None = None
     _groups: dict[str, list[str]] | None = None
 
+    def __repr__(self):

```

Comment:
could you have docstring and typing for both?

To explain exactly what you want to achieve with that?

Reply:
done